### PR TITLE
[PX-389] add githbub app pereqs to docs

### PR DESCRIPTION
**Pull request by kylemcnair**
Adding requirements for installing the Datafold GitHub integration

Screen Shot 2023-02-16 at 12 02 56 PM

Need to do the same for GitLab, but I'll do that separately

### CODE REVIEW

File: docs/integrations/git/github.md [hunk 1]
```diff
@@ -8,6 +8,12 @@ title: GitHub
 If you are on an on-prem deployment, you should first create a GitHub App for the integration. See [GitHub integration for Datafold on-prem](on-prem/content/github_on-prem.md) before proceeding with this tutorial.
 :::
 
+:::info Prerequisites
+* Your Datafold user must have admin permissions
```
Comment (leoebfolsom) [inline docs/integrations/git/github.md:12]: I would say "Datafold admin permissions"--redundant but this is confusing enough already

File: docs/integrations/git/github.md [hunk 2]
```diff
@@ -8,6 +8,12 @@ title: GitHub
...
+* Your GitHub account must be a member of the GitHub organization where the app is to be installed
```
Comment (leoebfolsom) [inline docs/integrations/git/github.md:13]: I would say "where the Datafold app is to be installed"

File: docs/integrations/git/github.md [hunk 3]
```diff
@@ -8,6 +8,12 @@ title: GitHub
...
+* Approval from a GitHub repo admin or GitHub organization owner is required
```
Comment (leoebfolsom) [inline docs/integrations/git/github.md:14]: "Approval of your request to add the Datafold app to your GitHub organization must be granted by a GitHub repo admin or GitHub organization owner"

**Review by leoebfolsom — APPROVED** [review]
Approved but I suggested some wording updates, take them or leave them 🙃

**Comment by kylemcnair** [comment]
added a tip for users who need GitHub admin help
Screen Shot 2023-02-16 at 5 00 57 PM

**Pull request merged**